ONSdigital/eq-author-app
# 1100 validator adding the other option 4

### What is the context of this PR?
Added validation for mandatory label on "Other" option.

### How to review 
1. Create at checkbox/radio answer with at least 2 options
2. Empty labels in options should have validation error and disappear when filled in. Error does not appear until input is blurred.
3. Ensure duplicate labels for options generate validation error
